examples/fluids/src/misc.c

```diff
@@ -397,7 +397,7 @@ PetscErrorCode RegisterLogEvents() {
 }
 
 // Print information about the given simulation run
-PetscErrorCode PrintRunInfo(User user, Physics phys_ctx, ProblemData problem, MPI_Comm comm) {
+PetscErrorCode PrintRunInfo(User user, Physics phys_ctx, ProblemData problem, TS ts, MPI_Comm comm) {
```

Comment:
We don't need `comm` if we already pass a `TS`. (There is also `user->comm`, which should match.)